vector_indexers/plot.py

```diff
@@ -0,0 +1,31 @@
+import matplotlib.pyplot as plt
+import json
+import pandas as pd
+
+
+def create_plot():
```

Comment:
In the benchmark is regularly used queries per second more than query time. Plus recall@1,5,... is an array as far as I remember. One need to choose which k to get. This can be added as parameter.

Reply:
How does this plot split between type of indexers? I do not see this logic here.

Reply:
In the results.json there's only faiss so I didnt remember to add the others. Ill do it now